python/tests/core/test_feature_monitoring_result_engine.py

```diff
@@ -309,8 +323,8 @@ def test_build_query_params_int(self):
         )
 
         # Act
-        query_params = result_engine.build_query_params(
-            start_time=start_time, end_time=end_time
+        query_params = result_engine._build_query_params(
+            start_time=start_time, end_time=end_time, with_statistics=False
```

Comment:
Please add a test setting with_statistics to true and checking that expand is in the query params